README.md

```diff
@@ -0,0 +1,70 @@
+# Analysis Schema
+
+A declarative schema and workflow framework for analysis and visualization of physical systems.
+
+The Analysis Schema creates a workflow system that processes a configuration file where users specify _what_ they would like the code to do, instead of _how_ to do it to complete analysis and visualization tasks. It supports high level analysis description that can be saved, shared, and reused in JSON document. 
+
+It consists of three pieces:
+
+1. A structured schema that guides the workflow. See the `yt_analysis_schema.json`. 
```

Comment:
Since this is in the readme, you can use [relative links](https://github.blog/2013-01-31-relative-links-in-markup-files/ )

```suggestion
1. A structured schema that guides the workflow. See [`yt_analysis_schema.json`](yt_analysis_schema.json). 
```

Reply:
got it!